integration-libs/punchout/components/punchout-session/punchout-session.component.ts

```diff
@@ -11,29 +11,51 @@ import {
   OnInit,
 } from '@angular/core';
 import { ActivatedRoute, Params } from '@angular/router';
+import { GlobalMessageService, GlobalMessageType } from '@spartacus/core';
 import { PUNCHOUT_SESSION_KEY, PunchoutFacade } from '@spartacus/punchout/root';
 import { switchMap, take } from 'rxjs';
 @Component({
   selector: 'cx-punchout-session',
-  template: `<p>Punchout session loading</p> `,
+  template: '',
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: false,
 })
 export class PunchoutSessionComponent implements OnInit {
   protected activatedRoute = inject(ActivatedRoute);
   protected punchoutFacade = inject(PunchoutFacade);
+  protected globalMessageService = inject(GlobalMessageService);
 
   ngOnInit(): void {
     this.activatedRoute.queryParams
       .pipe(
         take(1),
-        switchMap((param: Params) =>
-          this.punchoutFacade.getPunchoutSession({
+        switchMap((param: Params) => {
+          this.globalMessageService.add(
+            {
+              key: 'punchout.initiatingUserSession',
+            },
+            GlobalMessageType.MSG_TYPE_INFO
+          );
+          return this.punchoutFacade.getPunchoutSession({
             punchoutSessionId: param?.[PUNCHOUT_SESSION_KEY],
-          })
-        ),
+          });
+        }),
         take(1)
       )
-      .subscribe();
+      .subscribe({
+        error: (error) => {
```

Comment:
i think we can remove the full error callback as we andling the error (global message and redirection) in 
this.punchoutFacade.getPunchoutSession